crates/bitwarden-crypto/src/keys/master_key.rs

```diff
@@ -342,16 +359,21 @@ mod tests {
         let user_key: EncString = "0.8UClLa8IPE1iZT7chy5wzQ==|6PVfHnVk5S3XqEtQemnM5yb4JodxmPkkWzmDRdfyHtjORmvxqlLX40tBJZ+CKxQWmS8tpEB5w39rbgHg/gqs0haGdZG4cPbywsgGzxZ7uNI=".parse().unwrap();
 
         let decrypted = master_key.decrypt_user_key(user_key).unwrap();
+        let decrypted = if let SymmetricCryptoKey::Aes256CbcHmacKey(k) = &decrypted {
+            k
+        } else {
+            panic!("Decrypted key is not an Aes256CbcHmacKey");
+        };
```

Comment:
Using let-else should simplify this a bit:
```suggestion
        let SymmetricCryptoKey::Aes256CbcHmacKey(decrypted) = &decrypted else {
            panic!("Decrypted key is not an Aes256CbcHmacKey");
        };
```

Reply:
Nice, syntax I definitely was not aware of. Applied!